src/app/onboarding/views/AddNewServicesCode.vue

```diff
@@ -151,11 +151,11 @@ function clearTimeout(): void {
 }
 
 .status--is-connected {
-  color: var(--green-500);
+  color: var(--kui-color-text-success);
 }
 
 .status--is-disconnected {
-  color: var(--red-500);
+  color: var(--kui-color-text-danger);
```

Comment:
question: can you use the SCSS variables here instead?

Reply:
Fixed in https://github.com/kumahq/kuma-gui/pull/1314/commits/b5e11c5c05ecb42b15373dbad1068e27f5796288.